src/graphqlHelper.ts

```diff
@@ -1,19 +1,19 @@
-import axios from "axios";
+import axios from 'axios'
 
 export const sendQuery = (query: any, variables?: any): Promise<any> => {
-  return axios.post("http://localhost:4000/graphql?", {
-    query,
-  });
-};
+	return axios.post('http://localhost:4000/graphql?', {
+		query,
+	})
+}
 
 export const getAdsQuery = () => {
-  return `{
+	return `{
           ads{title, createdAt, id, username,comments{body, id, username}}
```

Comment:
add `user{firstName, lastName etc}` this way you can get all information required to display offer